### PR TITLE
Make running code cleanliness less painfull

### PR DESCRIPTION
**Pull request by cmorty**
* Only check for modifications in tracked files
* Tell the user to stash stuff, before running

### CODE REVIEW

**Comment by jonahgraham** [comment]
I had no idea people ran the scripts before commit and push :-)

File: releng/scripts/check_code_cleanliness.sh [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 set -e
 
+if test ! -z "$(git status -s -uno)"; then
+    echo "You have changes. Please stash them before continuing."
+fi
```
Comment (jonahgraham) [inline releng/scripts/check_code_cleanliness.sh:17]: Right after submitting I wondered if this should actually `exit 1`?

Comment (cmorty) [reply releng/scripts/check_code_cleanliness.sh:17]: OOps. That somehow got lost. Sorry :disappointed:

**Comment by cmorty** [comment]
@jonahgraham Moving all the patches to a local system by hand is a PITA. It's easier to just run them locally.

File: releng/scripts/check_code_cleanliness.sh (hunk 1 above)
Comment (cmorty) [reply releng/scripts/check_code_cleanliness.sh:17]: Actually not somehow. I removed it (and didn't comment it out :-( ) to be able to run the script on the other branch without committing it.

**Comment by jonahgraham** [comment]
> @jonahgraham Moving all the patches to a local system by hand is a PITA. It's easier to just run them locally.

OK - I am more than happy to receive any script updates that make them more useful to you!

**Pull request merged**